### PR TITLE
Correct inaccuracies in ECSQL Documentation

### PR DESCRIPTION
**Pull request by SpencerWBarnes**
Fix:
- Realworld -> real-world
- Parametrizing section references UserLabel
- SQL Functions section references the substring 'Fabric'

### CODE REVIEW

File: tools/build/tslint-rules/importWithinPackageRule.js [hunk 1]
```diff
@@ -0,0 +1,65 @@
+"use strict";
```
Comment (calebmshafer) [inline tools/build/tslint-rules/importWithinPackageRule.js:1]: Can you delete all of the `tools/build/tslint-rules` folder? Looks like you may have picked it up from a previous pull.

Comment (SpencerWBarnes) [reply tools/build/tslint-rules/importWithinPackageRule.js]: Deleted the `tools/build/tslint-rules` folder and its sub folders

**Comment by SpencerWBarnes** [comment]
One of the failed checks seems to be from the `rush cover` issue, and the other looks very similar. How should these failures be addressed?

**Comment by calebmshafer** [comment]
@SpencerWBarnes the `rush cover` is a known failure. I'm working on a fix now.

**Review by calebmshafer — APPROVED** [review]

**Comment by SpencerWBarnes** [comment]
@Mergifyio update

**Pull request merged**